### PR TITLE
Update tag line

### PR DESCRIPTION
**Pull request by SamBarker**
New tag line!

Screenshot 2024-02-19 at 14 42 45

I added Topic encryption to the suggested subtitle as that is the one thing we have today and we are rightly proud of it.

### CODE REVIEW

File: _layouts/index.html [hunk 1]
```diff
@@ -11,8 +11,8 @@
         <img src="{{site.baseurl}}/images/kroxylicious_logo_rgb_no_text.svg" class="d-block mx-lg-auto img-fluid" alt="Kroxylicious Mascot" width="700" height="500" loading="lazy">
       </div>
       <div class="col-lg-6">
-        <h1 class="fw-bold text-body-emphasis lh-1 mb-3">Kroxylicious transparently inject behaviour into your Apache Kafka<sup>&#174;</sup> cluster</h1>
-        <p class="lead">Solutions to cross-cutting concerns in areas including security, policy-enforcement, audit and much more.</p>
+        <h1 class="fw-bold text-body-emphasis lh-1 mb-3">Kroxylicious, the snappy open source proxy for Apache Kafka<sup>&#174;</sup></h1>
+        <p class="lead">Topic encryption, policy-enforcement, multitency, audit and much more.</p>
```
Comment (k-wall) [inline _layouts/index.html:15]: typo + hyphenation 
```suggestion
        <p class="lead">Topic encryption, policy-enforcement, multi-tenancy, audit and much more.</p>
```

**Comment by k-wall** [comment]
I think the buttons should take us to the overview and use-cases, rather than getting started and docs.  Those are the resources that a new arrival is likely to want first.

**Comment by SamBarker** [comment]
> I think the buttons should take us to the overview and use-cases, rather than getting started and docs. Those are the resources that a new arrival is likely to want first.

I'd contest it should be `use-cases` or `overview` and `quick starts`. I lean towards `use-cases` rather than `overview`.  
As you say those are the important resources for people wanting context but we want to get people up and running with as little fuss as possible. I am starting wonder if we should just move overview below the fold on the homepage? 

The docs are good landing point for people wanting details.

I actually think it renders nicely with three buttons so maybe cover each base

Screenshot 2024-02-22 at 14 37 36

**Comment by SamBarker** [comment]
I've opened https://github.com/kroxylicious/kroxylicious.github.io/pull/40 to take the button conversation further.

**Pull request merged**